src/goInstallTools.ts

```diff
@@ -144,9 +144,12 @@ export function installTools(missing: Tool[], goVersion: GoVersion): Promise<voi
 		return;
 	}
 
-	let installingMsg = `Installing ${missing.length} ${missing.length > 1 ? 'tools' : 'tool'} at ${toolsGopath}${
-		path.sep
-	}bin`;
+	let binpath = toolsGopath + path.sep + "bin";
+	if(envForTools['GOBIN'] != undefined && envForTools['GOBIN'] != ''){
+		binpath = "GOBIN path " + envForTools['GOBIN']
```

Comment:
If GOBIN is set, then we can essentially skip the code path https://github.com/microsoft/vscode-go/blob/8660153af2e24cd015fde26d3b222954e2cd2c32/src/goInstallTools.ts#L124-L145 right?

Reply:
We still have to set the current go path `envForTools['GOPATH'] = toolsGopath;` since envForTools is passed as options when `go get` child process is created. GOBIN is only setting the path where the executable should be installed. Right?

Reply:
Hmmm fair point